src/Mail/MailInterface.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace ZfrMail\Mail;
+
+/**
+ * Simple interface for a mail
+ *
+ * @author Michaël Gallego
+ */
+interface MailInterface
+{
+    /**
+     * Add a new "from" address and return a new mail
+     *
+     * @param  string $from
+     * @return MailInterface
+     */
+    public function withFrom(string $from): MailInterface;
```

Comment:
If you use `@return MailInterface` or `@return self`, PHPStorm will complain about the following statement because there's no `withTemplate()` method in `MailInterface`:
```php
(new TemplatedMail())->withFrom('daniel@danielgimenes.com.br')->withTemplate('foo/bar');
```
So, to make PHPStorm happy, you may use `@return static` instead :D